content/influxdb/v2.0/write-data/troubleshoot.md

```diff
@@ -68,12 +73,52 @@ If you notice data is missing in your bucket, do the following:
 - Verify the timestamps match the [precision parameter](/influxdb/v2.0/write-data/#timestamp-precision).
 - Minimize payload size and network errors by [optimizing writes](/influxdb/v2.0/write-data/best-practices/optimize-writes/) 
 
-### Troubleshoot rejected points
+### Review rejected points
 
 InfluxDB may reject points even if the HTTP request returns "Success".
-If some of your data did not write to the bucket, check for [field type](/influxdb/v2.0/reference/key-concepts/data-elements/#field-value) differences between the missing data point and other points that have the same [series](/influxdb/v2.0/reference/key-concepts/data-elements/#series).
-For example, did you attempt to write `string` data to an `int` field?
+To get a log of rejected data points, query the [`rejected_points` measurement](/influxdb/v2.0/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
```

Comment:
This feature isn't in 2.0 AFAIK. Is it part of 2.1? If so, these changes should be applied there, not in 2.0.